.globalconfig

```diff
@@ -0,0 +1,8 @@
+# Top level entry required to mark this as a global AnalyzerConfig file
+is_global = true
+
+#### Diagnostic configuration ####
+# https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-options
+# https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/categories
+dotnet_analyzer_diagnostic.category-reliability.severity = warning
+dotnet_analyzer_diagnostic.category-security.severity = warning
```

Comment:
Any other analyzer we would want to enable in the future?

Reply:
Sure, unfortunately enabling all of them at once is not a good idea because it would give us too many warnings.